MADE.App.Views.Controls.HeaderedTextBlock.Forms/HeaderedTextBlock.xaml

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Could the Xamarin.Forms component be created using a custom renderer for each platform instead of a XAML content view?

Reply:
This was a consideration. This wasn't done before however as it seemed overkill to create something with custom renders when it was simpler to create a content view.